bool enable
int32 degree
---
